packages/client/src/utils.js

```diff
@@ -106,3 +106,17 @@ export const getVaultId = (identifiers, coinType) => {
     (id) => id.indexOf(COIN_TYPE_TO_META[coinType].vaultName) >= 0
   );
 };
+
+export const showError = (modalContext, error) => {
```

Comment:
Let's encapsulate the showError and extractError into a `useErrorMessage` hook. Utils should not be aware of the components

Reply:
Updated, tnx for help!